core/smn_console.cpp

```diff
@@ -979,11 +979,15 @@ static cell_t sm_ServerCommandEx(IPluginContext *pContext, const cell_t *params)
 	}
 
 	engine->ServerExecute();
-
-	g_ShouldCatchSpew = true;
-	engine->ServerCommand("sm_conhook_start\n");
-	engine->ServerCommand(buffer);
-	engine->ServerCommand("sm_conhook_stop\n");
+	if (!g_ShouldCatchSpew) {
+		g_ShouldCatchSpew = true;
+		engine->ServerCommand("sm_conhook_start\n");
+		engine->ServerCommand(buffer);
+		engine->ServerCommand("sm_conhook_stop\n");
+	}
+	else {

```

Comment:
```suggestion
	} else {
```

Very minor nit, but prevailing style here is else on the same line as the closing brace.

Reply:
Fixed, thank you.